.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+---
+name: Bug report
+about: Create a report to help us to improve the project!
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Do we want to know (or ask) if SVN is also affected by the bug?

Reply:
To me that would be part of the "additional context" section but it might be clearer to explicitely mention it. The statement could be altered for eg:

"Add any other context about the problem you might found relevant (eg. screen capture, other Dosbox versions/forks impacted as well etc)."